src/modelize/modelize_property.nit

```diff
@@ -860,10 +864,8 @@ redef class AMethPropdef
 			assert mclassdef.default_init == null
 			mpropdef.is_old_style_init = is_old_style_init
 			mclassdef.default_init = mpropdef
-			if mpropdef.is_intro then
-				mpropdef.initializers.add mprop
-				mpropdef.is_calling_init = true
-			end
+			mpropdef.initializers.add mprop
```

Comment:
we add initializers only in intro. In refinements, new attributes are implicitly noautoinit (there is a warning)

Reply:
In this context you don't need to check mpropdef. Since the `mclassdef.is_intro` the mpropdef `default_init` is also is_intro. In this case I set the old_style_init mpropedef as an initializer. This point is need when a super class define this own old_style_init and inherited class use the default_init generated automaticlely.